docs/building-apps/build-properties.md

```diff
@@ -683,6 +683,28 @@ Consider using the unified [AppBundleResourcePrefix](#appbundleresourceprefix) p
 
 See also [IPhoneResourcePrefix](#iphoneresourceprefix) and [XamMacResourcePrefix](#xammacresourceprefix).
 
+## MonoUseCompressedInterfaceBitmap
+
+This directs the Mono runtime to use a compressed version of interface bitmaps
+(interface bitmaps are used to determine whether a certain types implements a
+given interface).
+
+These bitmaps can use a significant amount of memory at runtime, in particular
```

Comment:
I would remove the generic value types comment. The memory consumption scales pretty much with the number of interfaces that the application uses. Usage of the same interface with different generic types also counts.